NuRadioMC/SignalGen/ARZ/ARZ.py

```diff
@@ -468,6 +468,38 @@ def set_interpolation_factor2(self, interp_factor):
         """
         self._interp_factor2 = interp_factor
 
+    def get_shower_profile(self, shower_energy, shower_type, iN):
+        """
+        Gets a charge-excess profile
+
+        Parameters
+        ----------
+        shower_energy: float
+            the energy of the shower
+        shower_type: string (default "HAD")
+            type of shower, either "HAD" (hadronic), "EM" (electromagnetic) or "TAU" (tau lepton induced)
```

Comment:
Do we still store "TAU" shower profiles? I thought tau showers are handled differently nowadays

Reply:
This is just to be symmetric with `get_time_trace`

Reply:
But there is no keyword "TAU" in the shower library, or is there?

Reply:
Yes they exist.

Reply:
Ah I see what you mean.
I checked and they do not. I can pull this out in all docstrings